usr/sbin/regionsrv-enabler-azure

```diff
@@ -20,6 +20,25 @@ import sys
 
 import cloudregister.registerutils as utils
 
+CACHE_LICENSE_PATH = '/tmp/cached_license'
```

Comment:
The client already has a directory where it stores state, `REGISTRATION_DATA_DIR = '/var/lib/cloudregister/'` in utils. This would warrant a new function in the utils `get_state_dir()`.

Further if/when `/tmp` moves to tmpfs the information will be lost on reboot.